yt_dlp/extractor/rozhlas.py

```diff
@@ -51,7 +54,45 @@ def _real_extract(self, url):
         }
 
 
-class RozhlasVltavaIE(InfoExtractor):
+class RozhlasBaseIE(InfoExtractor):
+    _VALID_URL = None
+    _429_TIMEOUT = 1
+
+    def _extract_audio(self, entry, audio_id):
```

Comment:
```suggestion
    def _extract_formats(self, entry, audio_id):
```
Remove vars. `_VALID_URL` is not necessary in base class
Also, `_extract_formats` is the conventional name for this kind of method